jablib/src/main/java/org/jabref/logic/preferences/JabRefCliPreferences.java

```diff
@@ -715,6 +714,7 @@ public JabRefCliPreferences() {
         defaults.put(VERSION_IGNORED_UPDATE, "");
         defaults.put(VERSION_CHECK_ENABLED, Boolean.TRUE);
 
+        Localization.setLanguage(getLanguage());
```

Comment:
Why this was moved? Why not the comments?

I think, this should be kept as is?!?!?!